src/my.js

```diff
@@ -11,7 +11,7 @@ var g = require('./constants');
 var debug = process.env.DEBUG ? true : false;
 var sysfsFiles = {};
 
-exports.require = function (packageName, onfail) {
+require = function (packageName, onfail) {
```

Comment:
Do not redefine the global function 'require'.

Reply:
Sorry for the trouble ,i had it corrected. i will squash the commit